packages/common/src/interfaces.ts

```diff
@@ -209,4 +209,35 @@ export interface StateManagerInterface {
    */
   clearCaches(): void
   shallowCopy(downlevelCaches?: boolean): StateManagerInterface
+
+  /*
+   * Cache properties
+   */
+  _accountCache?: Cache
+  _storageCache?: Cache
+  _codeCache?: Cache
+
+  _accountCacheSettings?: CacheSettings
+  _storageCacheSettings?: CacheSettings
+  _codeCacheSettings?: CacheSettings
+}
+
+/**
+ * Cache related
+ */
+export enum CacheType {
+  LRU = 'lru',
+  ORDERED_MAP = 'ordered_map',
+}
+
+export type CacheSettings = {
+  deactivate: boolean
+  type: CacheType
+  size: number
+}
+
+interface Cache {
+  checkpoint(): void
+  commit(): void
+  revert(): void
```

Comment:
These `Common` interfaces should really only be used for the "3 big defining interfaces" (like StateManager) which are important beyond the scope of a single package and therefore should get some solid visibility and the possibility to remove.

(so generally it's already too crowded in here and some interfaces do *not* really belong here)

So these Cache interfaces should definitely not be placed here and should go back if it's not absolutely structurally necessary (I guess it's not?).

Reply:
(maybe this should be stated better at the top of this file)